samples/cache-sample/src/test/java/io/quarkiverse/infinispan/embedded/cache/sample/WeatherResourceTest.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkiverse.infinispan.embedded.cache.sample;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class WeatherResourceTest {
+
+    @Test
+    public void testWeatherEndpoint() throws Exception {
+        Weather weatherFirstCall = given()
+                .when().get("/weather/paris")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().as(Weather.class);
+
+        Weather weatherSecondCall = given()
+                .when().get("/weather/paris")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract().body().as(Weather.class);
+
+        assertThat(weatherFirstCall).isNotNull();
+        assertThat(weatherSecondCall).isNotNull();
+        assertThat(weatherFirstCall).isEqualTo(weatherSecondCall);
+
+        // For the example: force wait for override lifespan testing
+        Thread.sleep(1000);
```

Comment:
Maybe reduce to 500 ms? I don't know if there is a way to inject at time service to completely remove the delay which would preferred if possible.

Reply:
I'll put it to 500, I can do the injection of a time service but I can't use @Inject in a test that is then run in IT test mode too for native

Reply:
I used awaitability